tensorbay/client/version.py

```diff
@@ -582,3 +575,87 @@ def delete_tag(self, name: str) -> None:
         delete_data: Dict[str, Any] = {"name": name}
 
         self._client.open_api_do("DELETE", "tags", self._dataset_id, json=delete_data)
+
+
+class JobMixin:
+    """A mixin class supporting asynchronous jobs."""
+
+    _dataset_id: str
+    _client: Client
+    _status: Status
+
+    def _create_job(
+        self,
+        title: str,
+        job_type: str,
+        arguments: Dict[str, Any],
+        description: str = "",
+    ) -> str:
+        """Create a :class:`Job`.
+
+        Arguments:
+            title: The Job title.
+            job_type: The type of Job.
+            arguments: The arguments dict of the specific job.
+            description: The Job description.
+
+        Return:
+            The id of the job.
+
+        """
+
+    def _get_job(self, job_id: str) -> Dict[str, Any]:
+        """Get a :class:`Job`.
+
+        Arguments:
+            job_id: The Job id.
+
+        Return:
```

Comment:
`Return` or `Returns`

Reply:
Here we use `Return` to avoid error `DAR202` from flake8.

Reply:
It will be change in Implementation .